less/src/notify.less

```diff
@@ -26,11 +26,11 @@
         padding: @notify-padding-top 0 @notify-padding-bottom;
 
         @media all and (max-width: (@device-width-large + 20px)) {
-            padding: @notify-padding-top 20px @notify-padding-bottom;
+            padding: (@notify-padding-top / 2) 20px (@notify-padding-bottom / 2) !important;
         }
 
         @media all and (max-width: (@device-width-small + 20px)) {
-            padding: @notify-padding-top 10px @notify-padding-bottom;
+            padding: (@notify-padding-top / 3) 10px (@notify-padding-bottom / 3) !important;
         }
```

Comment:
The above two !important tags can be removed if the following snippet of CSS, located on line 15, is removed.

>.notify-type-prompt, 
    .notify-type-alert {
        .notify-popup-inner {
            padding: @notify-padding-top 0 @notify-padding-bottom;
        }
    }